daemon.py

```diff
@@ -154,7 +154,6 @@ def daemonize(self):
 
 
     def start(self, foreground=False):
-        log.info("Starting")
```

Comment:
I like this line, it lets you search easily in the logs for when the component started.